web/frontend/playwright.config.ts

```diff
@@ -1,5 +1,8 @@
 import { defineConfig, devices } from '@playwright/test';
 
+/* import environment variables used in development instance */
+require("dotenv").config({ path: "./../../.env" });
```

Comment:
```suggestion
require("dotenv").config({ path: "../../.env" });
```

Should work, shouldn't it?

Reply:
probably, but since for the imports it's also `./../module` I'm trying to stay coherent